requests/maniphest_search.go

```diff
@@ -0,0 +1,116 @@
+package requests
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/uber/gonduit/entities"
```

Comment:
Import group ordering used in the rest of the library suggests making this two groups:  standard library and the rest of the imports.

```suggestion

	"github.com/uber/gonduit/entities"
```